os/arch/xtensa/src/esp32/esp_err.h

```diff
@@ -16,6 +16,10 @@
  *
  ******************************************************************/
 
+#ifndef _ESP_ERR_H_
+#define _ESP_ERR_H_ 1
+
+#include "stdint.h"
```

Comment:
how about using `#include <stdint.h>` ?

Reply:
#define _ESP_ERR_H_ is enough

and besides, we can remove duplicate 'esp_err.h' in esp_idf_port, right?

Reply:
Agree with xixidodo :)